pkg/plugins/policies/core/matchers/dataplane.go

```diff
@@ -162,6 +162,12 @@ func dppSelectedByPolicy(
 			return inbounds, gwListeners, gateway, nil
 		}
 		return []core_rules.InboundListener{}, nil, false, nil
+	case common_api.Dataplane:
```

Comment:
I extracted the function that resolves targetRefs to `rules_common` package, I think it should be possible to use it for DPPs as it exactly the same approach (the change is not in `master` yet): https://github.com/kumahq/kuma/blob/5c48b20a6028770d9581b9ab2b81d9c782dfbf5a/pkg/plugins/policies/core/rules/common/resolvetargetref.go#L33